### ISSUE TITLE
[Suggestion] A forehead kiss

### ISSUE THREAD

**Issue by coder-incognito** [Monika-After-Story/MonikaModDev#8400]
Been doing more thinking on content for Monika After Story.
I've seen a recurring theme in real life relationships, not necessarily romantic even, where one kisses the forehead of the other.
It expresses appreciation, care, and a less 'romantically' and 'sexually' charged form of love between a couple. More so a softer love of adoration.
I think it would make sense to add to MAS, simply because going through the dialogue, there's several tender, sad, and even hopeful moments after which something like this might seem like the natural thing to do. Besides, it's a nice different addition to expression of affection. Don't get me wrong, the holding/hugging and kisses are certainly important and have their place, but I have a feeling that in some situations something like this would be more appropriate. Take for example saying goodbye. Sometimes a kiss is more appropriate, sometimes something like a forehead kiss would be more fitting.
As for the animation? Well here's an idea.
Pretty similar to the kiss animation, instead it just zooms towards her forehead. An appropriate expression would be one with her eyebrows bending towards her eyes (don't really know how to explain that but I hope you get what I mean with what eyebrows I refer to). Then when you've zoomed in quite close her eyes close, and so do yours for added effect and emphasis on this softer appreciation which this sort of expression is based on.
The button for it could say "Can I kiss your forehead?". Seems rather fitting.
Oh and, maybe the first time this happens, she gets teary eyed. I feel that would be appropriate for her character (thinking about various dialogue sequences where she gets teary eyed, like when you give her the ring).
As for affection levels... Well this could come at the same unlock as the actual kiss. The actual kiss has an effect of 'sealing the deal', so something to add to that to follow it up doesn't seem out of order.
That said, I wonder what the devs think about this.

**Comment by fromulus** [comment]
As cute as it sounds, I'm kind of against this, as multiple kiss types just for the sake of having multiple kiss types sound like useless bloat.

Keep in mind that we're never actually kissing or holding Monika, it's more like a silent agreement between the two of you to pretend doing it.
Because of that, I don't think either you or Monika would feel a difference compared to regular kisses (which are enough as it is imo, you can always imagine kissing her on the forehead/cheek /neck instead of the mouth since you're always just pretending anyway)

**Comment by coder-incognito** [comment]
> As cute as it sounds, I'm kind of against this, as multiple kiss types just for the sake of having multiple kiss types sound like useless bloat.
> 
> Keep in mind that we're never actually kissing or holding Monika, it's more like a silent agreement between the two of you to pretend doing it.
> Because of that, I don't think either you or Monika would feel a difference compared to regular kisses (which are enough as it is imo, you can always imagine kissing her on the forehead/cheek /neck instead of the mouth since you're always just pretending anyway). 

You do make a pretty valid point.
My counter point though is that while thinking about this, I thought about other kisses, like one's hand or whatever really, and they don't really seem as important in real life. I mean I'm basing what I state here off of what I've seen others do and what I've seen myself, and pretty much always the forehead kiss is kind of its own thing.
As for kissing anywhere since you're pretending. Again, good point, but take a look at her expression after you kiss her. Seems seem to indicate that it was a kiss that was a proper kiss as opposed to a kiss anywhere. Well sure, you could argue that the kiss could be somewhere else that the expression allows for, but for the forehead? That expression seems out of order I believe.
Other than that, the accompanying character development is a pro to me, the whole appreciation thing I mean.

**Comment by Booplicate** [comment]
I wouldn't mind that, but it raises a few questions.
Does it mean we'll add more kissing options in the future? You may not need them, but other people may find them important.
How do we handle it? You can't just kiss her, you have to ask first. But asking doesn't feel right. Neither is showing a menu with kissing options.

**Comment by coder-incognito** [comment]
> I wouldn't mind that, but it raises a few questions.
> Does it mean we'll add more kissing options in the future? You may not need them, but other people may find them important.
> How do we handle it? You can't just kiss her, you have to ask first. But asking doesn't feel right. Neither is showing a menu with kissing options.

Hmm. Well that's up to you devs, if you want to put it as a back burner thing, which you might add to if time allows for it. Can't really say much here because I can't speak for you devs 😅
Yeah the second thing does have me a bit perplexed. I'm thinking... The asking thing feels a bit out of order. How about adding it as a conditional? Meaning after you say goodbye an option pops up asking if you'd want to do that, kind of like the hold on option when you take her with you. Something like that after appropriate dialogue as well, like when she talks about not wanting to lose you to death? It'll make it a very reserved thing this way but the code complexity increases due to this integration with other dialogue and goodbyes.
And this still doesn't address the whole asking thing, yeah. I guess that's why I think making it accessible only after 400 affection makes sense, since after that point it can be said that you and her share a pretty comfortable bond.

**Comment by Booplicate** [comment]
> It'll make it a very reserved thing this way

I like that, but I'm wondering if we'd want it to be available all the time or not.

**Comment by coder-incognito** [comment]
> > It'll make it a very reserved thing this way
> 
> I like that, but I'm wondering if we'd want it to be available all the time or not.

Yeah that's a decision that needs to be taken with more opinions, I agree.

Here's an idea though, what if, in addition to those conditionals (the goodbyes and those topics, except the good night one since that's for a proper kiss) there's one more conditional. 
"Thanks for being there for me."
Seems like an appropriate topic. It's about appreciation and you can say it whenever you want after all. Also, it's unlocked at 400 affection, so it seems to be in line with the comfortability of the bond?

**Comment by ThePotatoGuy** [comment]
Yeah, a proactive kiss has some technical and immersion roadblocks to becoming a thing. leaving open in case of further discussion.

**Comment by coder-incognito** [comment]
> Yeah, a proactive kiss has some technical and immersion roadblocks to becoming a thing. leaving open in case of further discussion.

@ThePotatoGuy 

Hmm, well I do wonder what those roadblocks are.

I also didn't consider if it went against the project philosophy. I mean, I was thinking about it and I realized one thing. It's a bit funny how in order to interact with Monika, you rely on stuff like buttons and all, very 'procedural' stuff, right? And then Monika is also, at least this is how I think it's meant to be, reacting normally. You do see in a few topics her comments on reality and what is real and what is not after all. It's like a tables turn thing, isn't it? Now instead of her being trapped in a procedural thing, that is, DDLC, it's you, the player, as you try to communicate with her as she reacts in human ways! Talk about a twist!

So if that is how the philosophy is like, then it makes sense to not include something like this.
But if it's supposed to be as real as it gets, as she says for example in the window react for this repository, then I wonder. why not include it? Pretty immersive then, right? I wonder. What if like a first kiss, it's something that Monika initiates, by say some topic like kisses between people and their meanings. It sounds like a Monika dialogue I think. She could go on about platonic kisses, etc, and then she could mention the forehead kiss and it's place in a bond you share with someone. A little bit of dialogue as she mentions how it's common for goodbyes and comforting, as well as appreciative affection, and she asks you for one. Player gets a choice, and there we are, now it's a thing for goodbyes and that compliment.
